packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import ExpandCollapsePanel, { PanelTrigger } from '../ExpandCollapsePanel';
+import randomId from '../../utils/rndid';
+import { IconType } from '../Icon';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  className?: string;
+  children: React.ReactNode | React.ReactNode[];
+  props?: React.HTMLAttributes<HTMLDivElement>;
+}
+
+const Accordion = ({ className, children, ...props }: AccordionProps) => {
+  return (
+    <div
+      className={classNames(className ? className : '', 'Accordion__container')}
+      {...props}
+    >
+      {children}
+    </div>
+  );
+};
+
+interface AccordionContainerProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
If this is intended to be an extension of the `ExpandCollapsePanel` component, we should instead extend those props instead of recreating all of the props here.

```suggestion
interface AccordionContainerProps extends ExpandCollapsePanelProps {
```